ops_settings_docker.sample.yml

```diff
@@ -1,6 +1,6 @@
 ---
-attribution: "service by https://openrouteservice.org | data by http://srtm.csi.cgiar.org"
-coord_precision: 1e-6
+attribution: "internal service of propamap | data by http://srtm.csi.cgiar.org"
+coord_precision: 0.00083333333333333333
```

Comment:
This is the exact precision? If not, why is the use of 1e-6 dispensed with?

Reply:
I think 1e-6 is the old one, but let the author confirm it.